frontend/src/lib-common/utils/local-date.ts

```diff
@@ -10,3 +11,33 @@ import LocalDate from 'lib-common/local-date'
 export const getAge = (dateOfBirth: LocalDate): number => {
   return LocalDate.today().differenceInYears(dateOfBirth)
 }
+
+/**
+ *
+ * Groups a list of dates to list of consecutive date ranges
+ */
+export function groupDatesToRanges(dates: LocalDate[]): FiniteDateRange[] {
+  if (dates.length > 0) {
+    const sorted = dates.sort()
```

Comment:
`.sort()` mutates the input array, which will at some point lead to problems that are difficult to debug